Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2015-07-10 -- v3.0.3
+- Clean up unused `FIRMessaging_PROBER` macro.
+- Updated topic subscription warning.
```

Comment:
Corrected the deprecation warning when subscribing to or unsubscribing from an invalid topic.